File: java-codellama7B-FP-all/taskid-27_flip_case-gen1.java
import java.util.*;
import java.lang.reflect.*;
import org.javatuples.*;
import java.security.*;
import java.math.*;
import java.io.*;
import java.util.stream.*;
class Problem {
    // For a given string, flip lowercase characters to uppercase and uppercase to lowercase.
    // >>> flipCase(("Hello"))
    // ("hELLO")
    public static String flipCase(String string) {

        String result = string;
        for (int i = 0; i < string.length(); i++) {
            char c = string.charAt(i);
            if (Character.isLowerCase(c)) {
                result = string.replace(c, Character.toUpperCase(c));
            } else if (Character.isUpperCase(c)) {
                result = string.replace(c, Character.toLowerCase(c));
            }
        }
        return result;
    }
    public static void main(String[] args) {
    assert(flipCase(("")).equals(("")));
    assert(flipCase(("Hello!")).equals(("hELLO!")));
    assert(flipCase(("These violent delights have violent ends")).equals(("tHESE VIOLENT DELIGHTS HAVE VIOLENT ENDS")));
    }

}

File: py-codellama7B-AWQ-all/taskid-27_flip_case-gen1.py
def flip_case(string: str) -> str:
    """ For a given string, flip lowercase characters to uppercase and uppercase to lowercase.
    >>> flip_case('Hello')
    'hELLO'
    """

    result = ''
    for character in string:
        if character.islower():
            result += character.upper()
        elif character.isupper():
            result += character.lower()
        else:
            result += character
    return result
